app/src/main/res/values/strings.xml

```diff
@@ -825,6 +825,13 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_charging_station_operator_title">"Who is the operator of this charging station?"</string>
     <string name="quest_charging_station_capacity_title">"How many cars can be charged at this charging station at the same time?"</string>
 
+    <string name="team_mode">Team Mode</string>
+    <string name="team_mode_exit">Exit Team Mode</string>
+    <string name="team_mode_description">When mapping the same area together in a group, you can enter team mode to split up quests between all team members.</string>
+    <string name="team_mode_team_size_label">How many people are you?</string>
```

Comment:
Sorry but this is rather Germanic, one of these two is probably more English.
```suggestion
    <string name="team_mode_team_size_label">How many people are there?</string>
```
```suggestion
    <string name="team_mode_team_size_label">How many people in your team?</string>
```

Reply:
What about "How many are you?" I mean, what else?

Reply:
It still doesn't sound quite right to a native.

If you want something shorter:
```suggestion
    <string name="team_mode_team_size_label">How many people?</string>
```

Reply:
Or
```suggestion
    <string name="team_mode_team_size_label">How many of you are there?</string>
```

Reply:
https://ell.stackexchange.com/questions/181748/how-many-are-you-guys-or-how-many-people-are-you-guys-which-one-is-correct

Reply:
@westnordost Why did you change this back to the string I had initially?

Reply:
Oh, it's just that it appeared to be better in my ears, I didn't remember this discussion

Reply:
Fixed in #2613.